docs/sphinx/cookbook/move_to_managed.rst

```diff
@@ -0,0 +1,18 @@
+.. _move_to_managed:
+
+==================================
+Moving Host Data to Managed Memory
+==================================
+
+When using a system with NVIDIA GPUs, you may realize that some host data
+should be moved to unified memory in order to make it accessible by the GPU.
+You can do this with the :func:`umpire::ResourceManager::move` operation:
+
+.. literalinclude:: ../../../examples/cookbook/recipe_move_to_managed.cpp
+                    :lines: 34
+
+The move operation will copy the data from host memory to unified memory,
+allocated using the provided ``um_allocator``. The complete example is included
```

Comment:
I believe that this operation will also deallocate the memory used from the source of the data move.  I noticed that this is stated in the comment of the cookbook source, but perhaps it also merits being stated here?

Reply:
Good idea, I have addressed this